src/documents/articles/binary-operations-with-irregular-sets/index.html.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Binary operations with irregular sets
+
+date: 2015-09-15
+
+source:
+  name: Бинарные операции над неупорядоченными множествами
+  url: http://habrahabr.ru/post/248229/
+  lang: RU
+
+author:
+  name: Roman Mestsheryakov
+  site: https://connected.su
+  github: romkagolovadvayha
+
+translator:
+  name: Varya Stepanova
+  github: varya
+  twitter: varya_en
+  site: http://varya.me/
+
+meta:
+  desc: >
+
+---
+
+In this post I will make algorithmization of the array operations and describe the operations.
```

Comment:
I would suggest to use "do" instead of "make" in this case.

http://www.vocabulary.cl/Intermediate/Do_Make.htm